components/skeleton/SkeletonParagraph.tsx

```diff
@@ -11,7 +11,7 @@ export interface SkeletonParagraphProps {
   rows?: number;
 }
 
-class Paragraph extends React.Component<SkeletonParagraphProps, {}> {
+class SkeletonParagraph extends React.Component<SkeletonParagraphProps, {}> {
```

Comment:
Why change component name here?

Reply:
I think this paragraph class onlt for skeleton internal.  So rename it add skeleton prefix.

Reply:
Maybe all removed `Skeleton` prefix since current files name is not same style:
https://github.com/ant-design/ant-design/tree/52c8e837350fa654f484c963e096447516f74091/components/skeleton

Reply:
Got it. I will fix it tonight.

Reply:
fixed.